### DISCUSSION TITLE
Calling endPoint Contract Agreement V3 /v3/contractdefinitions/request

### DISCUSSION THREAD

**Discussion by JorgeBarto** [eclipse-edc/Connector#4437] [General]
Hi everyone,

after calling **satisfactorily** others **endPoints** such as: **createAsset**, **createPolicy** **createContract**... even getting an agreemenId after negociation. We are trying to use the subject endPoint (v3/contractdefinitions/request,) because we'd like to obtain all contract definitions (at least one we made before using /v3/contractdefinitions) but, getting **Http Error 405**.

Is there any extra needed lib or any extra needed configuration to add, in order to get a satisfactory answer, 200.

Thanks in advantage!

**Comment by ndr-brt** [comment]
HTTP error 405 means "method not allowed", and [`v3/contractdefinitions/request`, is a POST](https://eclipse-edc.github.io/Connector/openapi/management-api/#/Contract%20Definition%20V3/queryContractDefinitionsV3), please verify to use the correct "POST" method, and you should get some satisfaction :)

**Reply by JorgeBarto** [reply]
I'll let you know when I try ndr-brt. Thanks :-) !!

**Reply by JorgeBarto** [reply]
Thanks ndr-brt, now It run perfectly.
I appreciate the help!!
